src/theia/sfm/reconstruction_builder.cc

```diff
@@ -228,6 +228,11 @@ void ReconstructionBuilder::RemoveUncalibratedViews() {
   }
 }
 
+
+bool ReconstructionBuilder::SetMasksForFeaturesExtraction(std::vector<std::string> mask_filepaths) {
```

Comment:
The following line : 
 < return feature_extractor_and_matcher_->SetMasksForFeaturesExtraction(mask_filepaths); >
is > 80 characters wide. How do you manage this case?